docs/docsite/rst/dev_guide/developing_modules_checklist.rst

```diff
@@ -120,7 +120,7 @@ Read the complete :ref:`module metadata specification <ansible-metadata-block>`
 * When fetching URLs, please use either fetch_url or open_url from ansible.module_utils.urls 
   rather than urllib2; urllib2 does not natively verify TLS certificates and so is insecure for https. 
 * facts modules must return facts in the ansible_facts field of the result
-  dictionary. :ref:`module_provided_facts`
+  dictionary.
```

Comment:
I think this one should point here: reference_appendices/common_return_values.rst

Reply:
Done.